src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -36,6 +37,13 @@ p5.RendererGL.prototype.beginShape = function(mode) {
   return this;
 };
 
+const immediateBufferStrides = {
+  vertices: 1,
+  vertexNormals: 1,
+  vertexColors: 4,
+  uvs: 2
+};
```

Comment:
What do you think of writing unit tests for the other buffers here? When I throw in bad stride values for vertexColors, normals, and uvs the unit tests don't fail and the visual change on the manual test would only be apparent if I had seen them as a smooth gradient previously.

Reply:
That makes sense! I'll add tests for those too.